src/main/java/gov/cms/mat/cql_elm_translation/service/filters/CqlTranslatorExceptionFilter.java

```diff
@@ -111,4 +111,20 @@ private List<CqlCompilerException> filterBySyntax(
                     .contains("org.cqframework.cql.cql2elm.CqlSyntaxException"))
         .toList();
   }
+
+  /*
+   * MAT-7995: error: "No Viable Input at 'define :'"
+   * should be customized as: "Definition is missing a name."
+   * This is done in cql-antlr-parse, so on the frontend we don't want a duplicate error message
+   * therefore we are filtering it out here.
+   */
+  private List<CqlCompilerException> filterOutCustomErrors(
+      List<CqlCompilerException> filteredCqlTranslatorExceptions) {
+    return filteredCqlTranslatorExceptions.stream()
+        .filter(
+            cqlCompilerException ->
+                !"no viable alternative at input 'define :'"
+                    .equalsIgnoreCase(cqlCompilerException.getMessage()))
```

Comment:
How about checking for contains instead of equals? The presence of the colon leads me to believe the incoming string may have more text.

Reply:
@jkotanchik-SB just made the change, thanks!